server/src/infra/entities/album.entity.ts

```diff
@@ -52,4 +53,7 @@ export class AlbumEntity {
 
   @OneToMany(() => SharedLinkEntity, (link) => link.album)
   sharedLinks!: SharedLinkEntity[];
+
+  @DeleteDateColumn()
+  deletedAt?: Date;
```

Comment:
deletedAt!: Date | null

Reply:
Shouldn't the | null part be implicit because of the '?:' ?

Reply:
If you try to set this to null it will throw a type error, so no, it is not implied.